guides/common/modules/proc_using-a-third-party-repository.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Using_a_Third_Party_Repository_{context}"]
+= Using a Third Party Repository
+
+In this scenario, you use a third party repository to sync content.
+
+.On a connected server
+. In the connected server of the {ProjectWebUI}, navigate to *Content* > *Product*.
+. Click on the third party product.
+. Click over the third party repository.
```

Comment:
What does `over` mean in this context? perhaps `click on` or `click the` would be better.